utils/p4runtime_lib/convert.py

```diff
@@ -128,8 +139,15 @@ def encode(x, bitwidth):
     assert(not matchesIPv4('1000.0.0.1'))
     assert(not matchesIPv4('10001'))
 
+    assert(matchesIPv6('2001:0db8:85a3:0000:0000:8a2e:0370:7334'))
+    assert(not matchesIPv6('2001:0db8:85a3:0000:0000:8a2e:0370'))
+    assert(not matchesIPv6('2001:0db8:85a3:0000:0000:8a2e::7334'))
```

Comment:
Shouldn't this be a correct IPv6 address?  I do not understand yet why it would not be.

Reply:
By mistake, I added other test cases here, I'll review and modify it.